README.md

```diff
@@ -110,7 +110,109 @@ The last published version of the package become [available for download](htt
 
 ### Examples
 
-> **Will come soon**
+Below are spome examples of JavaScript IPFS in action.
+
+#### Startup (in Node.js or browser)
+
+There's still a bit of work required to start up a node in a robust way (i.e. without requiring the user to manually `js-ipfs init`). Here is one way to do it.
```

Comment:
This sentence might be confusing to new users. "What is starting a node in a robust way"? Why is doing 'init' not robust?